src/main/kotlin/calculator/domain/Operator.kt

```diff
@@ -0,0 +1,31 @@
+package calculator.domain
+
+import java.lang.IllegalArgumentException
```

Comment:
```suggestion
import kotlin.IllegalArgumentException
```
kotlin에서 제공하는 `IllegalArgumentException`이 존재합니다 :)